src/session/__tests__/sessionReducer-test.js

```diff
@@ -61,7 +62,7 @@ describe('sessionReducer', () => {
   });
 
   test('LOGIN_SUCCESS', () => {
-    const newState = sessionReducer(baseState, eg.action.login_success);
+    const newState = sessionReducer(baseState, actions.login_success);
```

Comment:
The object has many actions, but each one is a single example action. The name is chosen so that an example action of type `LOGIN_SUCCESS` is `eg.action.login_success`.

The pattern `eg.action.foo` is what appears all over the tests, so that's what the name is optimized for; we don't really ever use it as a collection.